test/helpers.jl

```diff
@@ -28,3 +28,15 @@ function ignore_output(body::Function; stdout::Bool=true, stderr::Bool=true)
 
     return result
 end
+
+# Alternatively in Julia 0.7.0-DEV.4517 we could use
```

Comment:
Aren't all the versions you use higher than this?

Reply:
They are. I've decided to leave this in as:
```julia
sprint(show, args..., context=:compact => true)
```
is a bit long compared to:
```julia
sprint(show_compact, args...)
```

Reply:
Maybe remove the "in Julia 0.7.0-DEV.4517" then? The comment currently makes it sound like you're planning to replace it.